stable/jenkins/ci/casc-values.yaml

```diff
@@ -2,10 +2,31 @@ master:
   JCasC:
     enabled: true
     defaultConfig: true
+    authorizationStrategy: |-
+      loggedInUsersCanDoAnything:
+        allowAnonymousRead: true
+    securityRealm: |-
+      ldap:
+        configurations:
+          - server: ldap.acme.com
+            rootDN: dc=acme,dc=uk
+            managerPasswordSecret: ${LDAP_PASSWORD}
+            groupMembershipStrategy:
+              fromUserRecord:
+                attributeName: "memberOf"
   sidecars:
     configAutoReload:
       enabled: true
+  enableXmlConfig: false
   healthProbeLivenessInitialDelay: 10
   healthProbeReadinessInitialDelay: 10
+  installPlugins:
+    - kubernetes:1.25.3
+    - workflow-job:2.38
+    - workflow-aggregator:2.6
+    - credentials-binding:1.21
+    - git:4.2.2
+    - configuration-as-code:1.39
+    - ldap:1.24
```

Comment:
It would be great if there was an option to add plugins without having to redefine the chart's default plugins. I opened #22497 for this.